service/replication.go

```diff
@@ -249,18 +279,18 @@ func (s *service) DeleteStorageProtectionGroup(ctx context.Context,
 	}
 	for key := range childs {
 		log.Info("Child Path: ", key)
-		export, err := isiConfig.isiSvc.GetExportWithPathAndZone(ctx, key, "")
+		_, err := isiConfig.isiSvc.GetExportWithPathAndZone(ctx, key, "")
 		if err == nil {
-			log.Error("Contains paths: ", export.Paths)
 			return nil, status.Errorf(codes.Internal, "VG is not empty")
```

Comment:
could help in future if we log the value of key/VG

Reply:
done